scripts/core/editor3/store/index.ts

```diff
@@ -137,9 +137,11 @@ export function onChange(contentState, {plainText = false} = {}) {
     let objectToUpdate = pathToValueArray.length < 2 ?
         this.item :
         pathToValueArray.slice(0, -1).reduce((obj, pathSegment) => {
-            const nextObj = obj[pathSegment];
+            if (obj[pathSegment] == null) {
+                obj[pathSegment] = {};
```

Comment:
What's wrong with null? :slightly_smiling_face:

Reply:
https://github.com/superdesk/superdesk-client-core/blob/ee37f7c142ef115938f40ea479fd7e6ab6658fa5/scripts/core/editor3/store/index.ts#L150
gives cannot read [key] of undefined if there is no `extra` field in a template.